### PR TITLE
Add performerByURL and movieByURL scrapers to Gamma Entertainment and Jules Jordan

### PR DESCRIPTION
**Pull request by woodgen**
This patch set adds performer scrapers to Gamma Entertainment and Jules Jordan.
GE was only tested with Evil Angel, but works well. I also just scrapes the photo and name, which is enough for my needs.

Jules Jordan currently gets the wrong height in imperial units. To fix this I am going to write a post process in go that converts that at some point.

### CODE REVIEW

**Comment by woodgen** [comment]
This now requires my `toMetric` post process which can be found here:
https://github.com/stashapp/stash/pull/711

**Comment by bnkai** [comment]
Adding the blocked label till 711 is sorted out.

**Comment by woodgen** [comment]
This is not blocked anymore, https://github.com/stashapp/stash/pull/711 was merged.

File: scrapers/JulesJordan.yml [hunk 1]
```diff
@@ -29,10 +43,54 @@ xPathScrapers:
...
+              with: https://www.julesjordan.com/
+
+  performerScraper:
+    performer:
+      Name: //span[@class='title_bar_hilite']/text()
+      Birthdate:
+        selector: //div[@class='model_bio model_bio_info']//text()[contains(.,'Age')]
+        postProcess:
+          - replace:
+            - regex: "Age:"
```
Comment (bnkai) [inline scrapers/JulesJordan.yml:60]: In some performers birthdate is missing and only the age eg `18` is present.
Not really much can be done about it though and constructing a regex to only match birth dates is probably a waste of time.

**Pull request merged**